synapse/eventbus.py

```diff
@@ -25,6 +54,26 @@ def __init__(self):
 
         self._fini_funcs = []
 
+        for name,valu in s_reflect.getItemLocals(self):
+
+            if not callable(valu):
+                continue
+
+            # check for onfini() decorator
+            if getattr(valu,'_ebus_onfini',None):
+                self.onfini(valu)
+                continue
+
+            # check for on() decorators
+            eons = getattr(valu,'_ebus_ons',None)
```

Comment:
We can do the following:

```
for name, filt in getattr(valu, '_ebus_ons', []):
    self.on(name, valu, **filt)
``` 

which allows us to skip the None check.  If we keep the current structure, make sure to change the none check to be identify instead of equality.